src/main/scala/Main.scala

```diff
@@ -33,12 +33,12 @@ object Main {
             }
               .map(_.foldLeft("") {
                   case (accumulator, (word, count)) =>
-                      accumulator + println( s"$word = $count\n")
+                      accumulator + s"$word = $count\n"
```

Comment:
However, every time string concatenation creates a new copy of the previous string. It's very slow! `O(N^2)`.
Wouldn't it be better to use `List[String]()` as an accumulator?